src/page-objects/administration/PaymentDetail.ts

```diff
@@ -0,0 +1,20 @@
+import type { Page, Locator } from '@playwright/test';
+import type { PageObject } from '../../types/PageObject';
+
+export class PaymentDetail implements PageObject {
+
+    public readonly header: Locator;
+    public readonly nameField: Locator;
+    public readonly availabilityRule: Locator;
+
+
+    constructor(public readonly page: Page) {
+        this.header = page.locator('.smart-bar__header');
+        this.nameField = page.getByLabel('Name').first();
+        this.availabilityRule = page.locator('.sw-settings-payment-detail__field-availability-rule').locator('.sw-entity-single-select__selection-text');
```

Comment:
mhh it is better to use here the following approach or?
```
this.ruleSelectionInput = page.locator('..sw-settings-payment-detail__field-availability-rule').locator('.sw-entity-single-select__selection');
        this.filtersResultPopoverSelectionList = page.locator('.sw-select-result-list__content').getByRole('listitem');
```

Reply:
Well, I don't think this improves the test much, but it sure was a nice coding exercise. However, it will add quite a lot of code for—if you ask me—no good reason. What do you think?